crates/test-programs/command-tests/src/bin/random.rs

```diff
@@ -1,6 +1,6 @@
 fn main() {
-    let mut byte = [0_u8];
-    getrandom::getrandom(&mut byte).unwrap();
+    let mut bytes = [0_u8; 256];
+    getrandom::getrandom(&mut bytes).unwrap();
 
-    assert_eq!(42, byte[0]);
+    assert!(bytes.iter().any(|x| *x != 0));
```

Comment:
Is there a way to either seed the RNG with some predefined value or provide a custom implementation of the `get_random` function to avoid assertions like this and instead assert a constant?